Mage.Server.Plugins/Mage.Player.Human/src/mage/player/human/HumanPlayer.java

```diff
@@ -2084,14 +2084,16 @@ public void selectBlockers(Ability source, Game game, UUID defendingPlayerId) {
         if (skipButtonActivated && !canStopOnAny && !canStopOnZero) {
             return;
         }
+        java.util.List<UUID> possibleBlockers = game.getBattlefield().getActivePermanents(filter, playerId, game).stream()
```

Comment:
It’s controlling by user settings above already (stop on zero blockers):
![IMG_1346](https://github.com/user-attachments/assets/349da27d-e5e6-44f1-9197-9aa6829824ac)